### DISCUSSION TITLE
Custom Naming

### DISCUSSION THREAD

**Discussion by FallenHoot** [Azure/avdaccelerator#349] [General]
A question about the custom naming. 

I understand the need for customization, but I don't understand the 100 lines of code needed for this. Wouldn't it be just as simple to have --- Here is the default, change if needed.

**Comment by danycontre** [comment]
@FallenHoot thanks for your feedback.

Agree we need to make some updates on naming and custom naming, especially for the custom portal UI (it is in the backlog). 

CC: @swathibhat1
